core/storage/beam_job/gae_models.py

```diff
@@ -126,11 +126,17 @@ def get_export_policy(cls):
 class BeamJobRunResultModel(base_models.BaseModel):
     """Represents the result of an Apache Beam job.
 
-    IDs are generated by Google Cloud Dataflow, not Oppia, and are guaranteed to
-    be unique:
-    https://cloud.google.com/dataflow/docs/reference/rest/v1b3/projects.jobs#resource:-job
+    IMPORTANT: Datastore entities have a size limit of ~1MiB (1,048,572 bytes,
+    to be precise). To support jobs that may need to produce outputs that exceed
+    this this limit, BeamJobRunResultModels are split into _unordered_ batches.
+    Thus, to return the full results you must fetch all models with matching
+    job_id values and concatenate their results.
+
+    The IDs of BeamJobRunResultModel are inconsequential, and only used as keys.
```

Comment:
Should still explain how they are generated -- do you want to keep the "IDs are generated by Google Cloud Dataflow" comment from the previous version of this code?

Reply:
Done.